diff -up linux-3.10.0-693.2.2.el7-org/include/net/netlabel.h linux-3.10.0-693.2.2.el7/include/net/netlabel.h 
--- linux-3.10.0-693.2.2.el7-org/include/net/netlabel.h 2017-09-27 15:08:04.855055715 +0200
+++ linux-3.10.0-693.2.2.el7/include/net/netlabel.h 2017-09-29 08:42:14.542064497 +0200
@@ -525,13 +525,6 @@ static inline int netlbl_catmap_setrng(s
 {
        return 0;
 }
-static int netlbl_catmap_setlong(struct netlbl_lsm_catmap **catmap,
-                                u32 offset,
-                                unsigned long bitmap,
-                                gfp_t flags)
-{
-       return 0;
-}
 static inline int netlbl_enabled(void)
 {
        return 0;
